requirements-test.txt

```diff
@@ -5,3 +5,4 @@ black==20.8b1
 isort==5.6.*
 flake8==3.8.*
 mypy==0.790
+freezegun==1.0.*
```

Comment:
instead of using `freezegun`, could you allow `Client.set_jwt_header()` to optionally accept the datetime?

Reply:
What'd be the advantage of adding the parameter to `Client.set_jwt_header()` vs using `freezegun`. I can't see a case where one should define  the date manually except for Unit tests.